mathics/builtin/datentime.py

```diff
@@ -549,10 +566,22 @@ class TimeZone(Predefined):
      = ...
     """
 
-    name = '$TimeZone'
+    name = "$TimeZone"
+    value = Real(-time.timezone / 3600.)
```

Comment:
We should probably avoid duplicating code in here. We could make `value` a static field of `SystemTimeZone` and simply clone `SystemTimeZone.value` in here.

Reply:
Ok. Would you just then make the changes and commit over this. Thanks!

Reply:
Sure. I'll work on it.